internal/pkg/term/selector/selector.go

```diff
@@ -863,33 +843,41 @@ var OnlyInitializedWorkloads WorkloadSelectOption = func(s *LocalWorkloadSelecto
 	s.onlyInitializedWorkloads = true
 }
 
-// Workload fetches all jobs and services in a workspace and prompts the user to select one.
-// It can optionally select only initialized workloads which exist in the app (default behavior)
-// or list all workloads for which there are manifests in the workspace.
-// The latter behavior can be specified by passing the LocalWorkloads
-func (s *LocalWorkloadSelector) Workload(msg, help string) (wl string, err error) {
-	summary, err := s.ws.Summary()
+// Workloads fetches all jobs and services in a workspace and prompts the user to select one or more.
+// It can optionally select only initialized workloads which exist in the app (by passing the
+// OnlyInitializedWorkloads option to NewLocalWorkloadSelector) or list all workloads for which
+// there are manifests in the workspace (default).
+func (s *LocalWorkloadSelector) Workloads(msg, help string) ([]string, error) {
```

Comment:
Instead of making `OnlyInitializedWorkloads` a part of the `selector` struct, I wonder if it makes sense to pass this option as an optional parameter to `Workloads`.

```go
type SelectWorloadsOption {
  OnlyInitialized bool
}
func (s *LocalWorkloadSelector) Workloads(msg, help string, options *SelectWorloadsOption) ([]string, error) 
```

This way we won't have to modify an existing selector / create a new selector to call the same `Workloads` function, just because the option has changed.

Realistically this is probably not a concern - I'm not aware of any existing codes in the code base that need to call `Workloads` two times in one execution cycle:
```go
s := &LocalWorkloadSelector{}
s.Workloads("", "") // Need initialized workloads only.
/*...*/
s.Workloads("", "") // Need initialized and non-initialized workloads.

```